include/dca/phys/parameters/mci_parameters.hpp

```diff
@@ -107,6 +110,10 @@ class MciParameters {
     return store_configuration_;
   }
 
+protected:
+  // Resize vector arguments to have the same size as the number of iterations.
+  void inline solveDcaIterationConflict(int iterations);
```

Comment:
Is this correcting for an input that is conflicting?  If so I'd rather see the application halt informing the user of the conflict.  If yes please write a issue so we go fix I guess mci_parameters outside of this PR.

Reply:
I want the user to either pass a vector that specifies the measurements for each iteration, or a scalar that is applied for each iteration. I don't really need to fix vectors of the wrong size, even if it is rather natural to cut excessive values or duplicate the last number of measurements (any sane simulation has a non decreasing number of measurements).

Reply:
Opened issue #214